torch/_meta_registrations.py

```diff
@@ -2252,6 +2252,10 @@ def _formula_transposed(ln: int, p: int, d: int, k: int, s: int, op: int) -> int
             ret_shape.append(
                 _formula(dims[i], padding[i], dilation[i], kernel_size[i], stride[i])
             )
+    torch._check(
+        any(x > 0 for x in ret_shape[2:]),
```

Comment:
Is it possible any of the size here to be -1?

Reply:
I don't think any tensor should have negative size.

https://github.com/pytorch/pytorch/blob/e1622dca7a727fe87c0adf5bebb2ae6363d2869e/aten/src/ATen/native/TensorFactories.cpp#L401
https://github.com/pytorch/pytorch/blob/e1622dca7a727fe87c0adf5bebb2ae6363d2869e/aten/src/ATen/EmptyTensor.h#L6-L15